tasks/nunjucks.js

```diff
@@ -35,7 +35,7 @@ module.exports = function(grunt) {
 
         async.each(this.files, function(f, done) {
             var filepath = path.join(process.cwd(), f.src[0]);
-            var data = options.data || {};
+            var data = JSON.parse(JSON.stringify(options.data || {}));
```

Comment:
Thanks a lot for fixing!

To be honest, I'd never find it out.

Maybe it's better to leave explaining comment before that line? parsing of stringified json to achieve cloned data may be unclear for some people.